taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/T2CAppiumUtils.java

```diff
@@ -25,28 +26,14 @@ public class T2CAppiumUtils {
     private static boolean isSelfTesting = false;
 
     public static WebElement findElement(BaseDriverController driver, BaseElementInfo element, Logger logger) {
-        WebElement elementFinded = null;
+        WebElement elementFound = null;
```

Comment:
(you can ignore) Not necessary for renaming here.

Reply:
Fix typo xD